myokit/formats/sbml/_importer.py

```diff
@@ -159,42 +188,101 @@ def model(self, path):
         # Return finished model
         return model
 
-    def _parse_initial_assignments(self, model, comp, refs, node):
+    def _get_namespaces(self, sbml_version: str, log):
```

Comment:
Myokit is used on Python 2 and Python 3, so `sbml_version : str` will have to become just `sbml_version`, I'm afraid